performance/software.md

```diff
@@ -467,13 +471,51 @@ https://docs.nvidia.com/deeplearning/performance/dl-performance-fully-connected/
 
 For parameters that are small, there is also [Dimension Quantization Effects](https://docs.nvidia.com/deeplearning/performance/dl-performance-matrix-multiplication/index.html#dim-quantization) to consider, this is where tiling happens and the right multiplier can have a significant speedup.
 
-
+The work in https://arxiv.org/abs/2401.14489 provides much greater detail on tile/wave quantization and the number of attention heads, but the highlights are:
 
 ### Tile and wave quantization
 
-XXX
+<img src="images/notation.png" width=20% height=20% />
```

Comment:
These images render too tiny in the markdown, at least I couldn't read them

Should we leave them as they are, as this is not a paper and there is no need to cram images into tiny columns as it's done in papers, which can't be read.

Let's just use?
```suggestion
![notation](images/notation.png)
```

Reply:
I can just go over those and make them a bit smaller if they are indeed too big. I am likely to have time tomorrow night to adjust those, I will take care of those. You did the heavy lifting already, Quentin.